content/docs/examples/advanced-gateways/egress-gateway/index.md

```diff
@@ -42,15 +51,15 @@ Another use case is a cluster where the application nodes do not have public IPs
     Note that any pod that you can `exec` and `curl` from would do.
 
 *   Create a shell variable to hold the name of the source pod for sending requests to external services.
-If we used the [sleep]({{<github_tree>}}/samples/sleep) sample, we run:
+    If you used the [sleep]({{<github_tree>}}/samples/sleep) sample, run:
 
     {{< text bash >}}
     $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
     {{< /text >}}
 
-## Define an egress `Gateway` and direct HTTP traffic through it
+## Define an egress gateway for HTTP traffic
```

Comment:
A small remark: you do not only define a gateway here, you also direct the traffic thru it. Maybe the precision should be satisfied for brevity here.

Reply:
Good point, but since I'm trying to keep the section titles short and sweet, a better approach might even be to remove the "Define an" part, since the example title is "﻿﻿Configure an Egress Gateway" and this section is simply doing that for HTTP traffic.